repository_service_tuf_api/bootstrap.py

```diff
@@ -223,21 +205,21 @@ def post_bootstrap(payload):
     logging.debug("Saving settings")
     for rolename, role_settings in payload.settings.roles.items():
         rolename = rolename.upper()
+        threshold = 1
+        num_of_keys = 1
```

Comment:
For non-root roles `threshold` and `num_of_keys` is 1 as we are going to use a single online key.